### DISCUSSION TITLE
Mobile Native dApps

### DISCUSSION THREAD

**Discussion by ghost** [MobileNativeFoundation/discussions#25] [General]
Just starting this thread in case anyone have interest in this topic.

Does anyone has good resources on the current state of mobile native dApps (and/or libraries to implement them)?
A lot of info I find online is already a couple years old.

Feel free to share any content related with it. 👍

**Comment by foobra** [comment]
what is d for?

**Reply by Armenm** [reply]
d stands for Decentralized 
https://ethereum.org/en/dapps/
https://en.wikipedia.org/wiki/Decentralized_application
